frontend/resources/styles/common/refactor/reorder.scss

```diff
@@ -0,0 +1,36 @@
+.reorder {
+  cursor: grab;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  left: var(--reorder-left-position, calc(-1 * $s-16));
+  z-index: $z-index-2;
+  position: absolute;
+  top: calc(-1 * $s-6);
```

Comment:
Please use the tokens defined in the design system (for instance in `spacing.scss`) and use the semantic ones (i.e. `--sp-l` when available).